tests/album-create.test.js

```diff
@@ -0,0 +1,48 @@
+// tests/album-create.js
+const { expect } = require('chai');
+const request = require('supertest');
+const app = require('../src/app');
+const getDb = require('../src/services/db');
+
+describe('create album', () =>{
+    let db;
+    let artists;
+    beforeEach(async () => {
+        db = await getDb()
+
+        await Promise.all([
+            db.query('INSERT INTO Artist (name, genre) VALUES (?, ?)', [
+                'Tame Impala',
+                'rock',
+            ])
+        ]);
+
+        [artists] = await db.query('SELECT * FROM Artist');
+    });
+
+    afterEach(async () => {
+        await db.query('DELETE FROM Album');
+        await db.close();
+    });
+
+    describe('/album', () => {
+        describe('POST', () =>{
+            it('creates a new album in the database', async () => {
+                const res = await request(app).post('/album').send({
+                    name: 'The Slow Rush',
+                    year: 2020,
+                    artistId: artists[0].id,
+                });
+                expect(res.status).to.equal(201);
```

Comment:
Could this be moved to be with the other assertions at the end? Generally the pattern for structuring tests is Arrange - Act - Assert: Arrange the variables/components needed - Act on them so they have been manipulated as they would be when in real use - Assert the behaviour is as you expect